bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/network/CULNetworkHandlerImpl.java

```diff
@@ -31,58 +37,35 @@
  * (CUN for example).
  *
  * @author Markus Heberling
+ * @author Till Klocke
  * @since 1.5.0
  */
-public class CULNetworkHandlerImpl extends AbstractCULHandler<CULNetworkConfig> {
+public class CULNetworkHandlerImpl extends AbstractCULHandler<CULNetworkConfig>implements Runnable {
```

Comment:
a lack of space before `implements` suggests the Eclipse IDE formatting wasn't applied or the settings are incorrect.